binding/Binding/Skottie/InvalidationController.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.ComponentModel;
+using System.IO;
+using System.Runtime.InteropServices;
+
+namespace SkiaSharp.Skottie
+{
+	public unsafe class InvalidationController : SKObject
+	{
+		public InvalidationController ()
+			: this (IntPtr.Zero, true)
+		{
+			Handle = SkiaApi.sksg_invalidation_controller_new ();
```

Comment:
This can be assigned in the ctor directly. Not sure if there was an optimisation, but it skips a few checks.